lib/common.js

```diff
@@ -161,13 +161,21 @@ var restoreOverriddenRequests = function() {
   requestOverride = [];
 };
 
+/**
+ * Get high level information about request as string
+ * @param  {Object} options
+ * @param  {string} options.method
+ * @param  {string} options.port
+ * @param  {string} options.proto
+ * @param  {string} options.hostname
+ * @param  {string} options.path
+ * @param  {Object} options.headers
+ * @param  {string|object} body
```

Comment:
A URL could also have a:
1. Query String `?a=123&b=456`
2. Fragment eg: `#00h02m01s`

https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL

Importance: 
Query String: Major, as we match on this
Fragment: Minor, I'm not sure Nock supports this